docs/interaction_based_testing.adoc

```diff
@@ -681,6 +681,18 @@ subscriber.receive(_) >>> ["ok", "fail", "ok"] >> { throw new InternalError() }
 This will return `"ok", "fail", "ok"` for the first three invocations, throw `InternalError`
 for the fourth invocations, and return `ok` for any further invocation.
 
+
+=== Returning a default response
+
+If you don't really care what you return, but you must return a non-null value, you can use `_`.
```

Comment:
It is not non-`null` in all cases, is it?
I mean to remember having seen in the code that under some circumstances it indeed returns `null`.

Reply:
Hm, maybe it was only for `void` and `Void` return types.

Reply:
Yes `org.spockframework.mock.EmptyOrDummyResponse` only returns `null` for `void` methods.